src/test/java/edu/cornell/kfs/sys/service/impl/AwsSecretServiceImplIntegrationTest.java

```diff
@@ -39,6 +45,17 @@
 
     @BeforeEach
     void setUp() throws Exception {
+        org.apache.log4j.LogManager.getLogger(AwsSecretServiceImpl.class).setLevel(org.apache.log4j.Level.DEBUG);
+        
+        /* 
+         * This logging was helpful in debugging things.  we don't want this logging long term, but it would be helpful to 
+         * keep them to be easily recovered
```

Comment:
Is there a better way to structure or explain the helper logging, so that we don't have to keep commented-out code around in our project?

Reply:
I think I found a better way